root/etc/systemd/system/discovery-autostart.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
 Description=Start downloaded extension scripts
+Requires=nm-connected.service
```

Comment:
Why this? We do not need to activate `nm-connected` when autostart (and others) are activated. I don't follow why we need this:

```
       Requires=
           Configures requirement dependencies on other units. If this unit gets activated, the units listed here will be
           activated as well. If one of the other units gets deactivated or its activation fails, this unit will be
           deactivated. This option may be specified more than once or multiple space-separated units may be specified in
           one option in which case requirement dependencies for all listed names will be created. Note that requirement
           dependencies do not influence the order in which services are started or stopped. This has to be configured
           independently with the After= or Before= options. If a unit foo.service requires a unit bar.service as
           configured with Requires= and no ordering is configured with After= or Before=, then both units will be started
           simultaneously and without any delay between them if foo.service is activated. Often it is a better choice to
           use Wants= instead of Requires= in order to achieve a system that is more robust when dealing with failing
           services.

           Note that dependencies of this type may also be configured outside of the unit configuration file by adding a
           symlink to a .requires/ directory accompanying the unit file. For details see above.
```